### PR TITLE
Upgrade to babel 7

### PR DESCRIPTION
**Pull request by nixterrimus**
Upgrades this project to babel 7. This is a prerequisite for migrating to jest.

### CODE REVIEW

File: src/components/app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ class App extends React.Component {
         keypadType: consts.KeypadTypes.EXPRESSION,
     };
 
-    handleChange = (e: SyntheticEvent<>) => {
```
Comment (kevinbarabash) [inline src/components/app.js]: How was this even working before? lol

File: webpack.config.externals.js [hunk 2]
```diff
@@ -23,9 +23,8 @@ module.exports = {
                 loader: 'babel',
                 query: {
                     presets: [
-                        ["es2015", {"loose": true}],
-                        "react",
-                        "stage-2",
+                        "@babel/env",
```
Comment (kevinbarabash) [inline webpack.config.externals.js]: Can you add a .browserslistrc that mirrors what's in https://github.com/Khan/webapp/blob/master/.browserslistrc?  The `@babel/env` preset will look for that file and configure itself based on the entries in that file.

File: src/components/app.js (hunk 1 above)
Comment (nixterrimus) [reply src/components/app.js]: Right? This should not have worked!

File: webpack.config.externals.js (hunk 2 above)
Comment (nixterrimus) [reply webpack.config.externals.js]: Done!

File: native.html [hunk 3]
```diff
@@ -12,18 +13,18 @@
...
 <body>
 
-<div id="root"></div>
+    <div id="root"></div>
 
-<script src="lib/i18n.js"></script>
-<script src="lib/icu-slim.js"></script>
-<script src="node_modules/jquery/dist/jquery.min.js"></script>
-<script src="mathquill/mathquill.js"></script>
-<script src="node_modules/babel-polyfill/browser.js"></script>
```
Comment (nixterrimus) [inline native.html]: I am nervous about removing this @kevinbarabash. Everything seems to work but there also doesn't seem to be a replacement. Does that seem right to you?

**Pull request merged**